core/templates/dev/head/pages/exploration_player/InputResponsePairDirective.js

```diff
@@ -25,18 +25,23 @@ oppia.directive('inputResponsePair', [
         oppiaAvatarImageUrl: '&',
         profilePicture: '&',
         getInputResponsePairId: '&inputResponsePairId',
+        last: '=',
```

Comment:
isLastPair perhaps? (And use & -- one-way binding is better if possible)

Reply:
doneaa